src/Avalonia.Controls/Platform/X11Properties.cs

```diff
@@ -0,0 +1,35 @@
+using Avalonia.Controls.Platform;
+using Avalonia.Platform;
+using Avalonia.Reactive;
+
+namespace Avalonia.Controls;
+
+public class X11Properties
```

Comment:
```suggestion
/// <summary>
/// Set of X11 specific properties and events that allow deeper customization of the application per platform.
/// </summary>
public class X11Properties
```

Not sure about attached properties documentation, can be copypasted from X11 docs or something.